### PR TITLE
Replace Code Hotspots tab screenshots and references

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do? What is the motivation?

* Code Hotspots tab renamed to Profiles.
* Breakdown view removed; only view is _timeline_ view now.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/profiler/enabling/java.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Minimum JDK versions:
 
 Because non-LTS JDK versions may not contain stability and performance fixes related to the Datadog Profiler library, use versions 8, 11, and 17 of the Long Term Support JDK.
 
-Additional requirements for profiling [Code Hotspots][12]:
+Additional requirements for profiling [Trace to Profiling integration][12]:
  - OpenJDK 11+ and `dd-trace-java` version 0.65.0+
  - OpenJDK 8 8u282+ and `dd-trace-java` version 0.77.0+
```
Comment (PyvesB) [inline content/en/profiler/enabling/java.md:61]:
```suggestion
 - OpenJDK 17.0.5+ and `dd-trace-java` version 1.17.0+
 - OpenJDK 11.0.17+ and `dd-trace-java` version 1.17.0+
 - OpenJDK 8 8u352+ and `dd-trace-java` version 1.17.0+
 - OpenJ9 17.0.6+ and `dd-trace-java` version 1.17.0+
 - OpenJ9 11.0.18+ and `dd-trace-java` version 1.17.0+
 - OpenJ9 8.0.362+ and `dd-trace-java` version 1.17.0+
```

File: content/en/profiler/enabling/php.md [hunk 2]
```diff
@@ -54,11 +54,11 @@ apk add libgcc
...
 The following profiling features are available in the following minimum versions of the `dd-trace-php` library:
 
-|      Feature              | Required `dd-trace-php` version          |
-|---------------------------|------------------------------------------|
-| [Code Hotspots][12]       | 0.71+                                    |
-| [Endpoint Profiling][13]  | 0.79.0+                                  |
-| [Timeline][15]            | 0.98.0+            |
+| Feature                              | Required `dd-trace-php` version |
+|--------------------------------------|---------------------------------|
+| [Trace to Profiling integration][12] | 0.71+                           |
```
Comment (PyvesB) [inline content/en/profiler/enabling/php.md:59]:
```suggestion
| [Trace to Profiling integration][12] | 0.89.0+                           |
```

File: content/en/profiler/enabling/python.md [hunk 3]
```diff
@@ -39,11 +39,11 @@ The installation requires pip version 18 or above.
...
 The following profiling features are available in the following minimum versions of the `dd-trace-py` library:
 
-| Feature                  | Required `dd-trace-py` version |
-|--------------------------|--------------------------------|
-| [Code Hotspots][12]      | 0.44.0+                        |
-| [Endpoint Profiling][13] | 0.54.0+                        |
-| [Timeline][15]           | 2.12.0+, 2.11.4+, or 2.10.7+   |
+| Feature                              | Required `dd-trace-py` version |
+|--------------------------------------|--------------------------------|
+| [Trace to Profiling integration][12] | 0.44.0+                        |
```
Comment (PyvesB) [inline content/en/profiler/enabling/python.md:44]:
```suggestion
| [Trace to Profiling integration][12] | 2.12.0+, 2.11.4+, or 2.10.7+ |
```

File: content/en/profiler/enabling/go.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ For a summary of the minimum and recommended runtime and tracer versions across
 
 The Datadog Profiler requires Go 1.19+.
 
-For [Code Hotspots][2] and [Endpoint Profiling][3], use `dd-trace-go` version 1.37.0+.
+For the [Trace to Profiling integration][2] and [Endpoint Profiling][3], use `dd-trace-go` version 1.37.0+.
```
Comment (PyvesB) [inline content/en/profiler/enabling/go.md:28]:
```suggestion
For the [Trace to Profiling integration][2] and [Endpoint Profiling][3], use `dd-trace-go` version 1.51.0+.
```

File: content/en/profiler/enabling/dotnet.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ The following profiling features are available in the following minimum versions
 | Allocations profiling     | Preview, 2.18.0+                   | .NET Framework beta (requires Datadog Agent 7.51+) and .NET 6+                           |
 | Lock Contention profiling | 2.49.0+                            | .NET Framework beta (requires Datadog Agent 7.51+) and .NET 5+                           |
 | Live heap profiling       | Preview, 2.22.0+                   | .NET 7+                                                                                  |
-| [Code Hotspots][12]       | 2.7.0+                             | All supported runtime versions.                                                          |
+| [Trace to Profiling integration][12]       | 2.7.0+                             | All supported runtime versions.                                                          |
```
Comment (PyvesB) [inline content/en/profiler/enabling/dotnet.md:66]:
```suggestion
| [Trace to Profiling integration][12]       | 2.30.0+                            | All supported runtime versions.                                                          |
```

File: content/en/profiler/enabling/supported_versions.md [hunk 6]
```diff
@@ -45,7 +45,7 @@ To access additional profiling features, use at least the minimum versions summa
 
 |                                   | [Java][1]  | [Python][2]  |  [Go][3]   |  [Ruby][4] |   [Node.js][5]  |  [.NET][6]   |   [PHP][7]  | [Rust/C/C++][8] |
 |-----------------------------------|:-------:|:-------:|:------------:|:------:|:---------:|:-------:|:------:|:----------:|
-| {{< ci-details title="Code Hotspots" >}}Find specific lines of code related to performance issues. <a href="/profiler/connect_traces_and_profiles/#identify-code-hotspots-in-slow-traces">Learn more</a>{{< /ci-details >}}   | [JDK&nbsp;8+][17] | tracer&nbsp;0.44.0 | tracer&nbsp;1.37.0 | tracer&nbsp;0.48.0 | tracer&nbsp;5.0.0,<br>4.24.0, 3.45.0 | tracer&nbsp;2.7.0 | tracer&nbsp;0.71.0 |      |
+| {{< ci-details title="Trace to Profiling integration" >}}Find specific lines of code related to performance issues. <a href="/profiler/connect_traces_and_profiles/#identify-code-hotspots-in-slow-traces">Learn more</a>{{< /ci-details >}}   | [JDK&nbsp;8+][17] | tracer&nbsp;0.44.0 | tracer&nbsp;1.37.0 | tracer&nbsp;0.48.0 | tracer&nbsp;5.0.0,<br>4.24.0, 3.45.0 | tracer&nbsp;2.7.0 | tracer&nbsp;0.71.0 |      |
```
Comment (PyvesB) [inline content/en/profiler/enabling/supported_versions.md:48]:
```suggestion
| {{< ci-details title="Trace to Profiling integration" >}}Find specific lines of code related to performance issues. <a href="/profiler/connect_traces_and_profiles/#identify-code-hotspots-in-slow-traces">Learn more</a>{{< /ci-details >}}   | [JDK&nbsp;8+][17] | tracer&nbsp;2.12.0,<br>2.11.4, 2.10.7 | tracer&nbsp;1.51.0 | tracer&nbsp;1.21.1 | tracer&nbsp;5.11.0,<br>4.35.0, 3.56.0 | tracer&nbsp;2.30.0 | tracer&nbsp;2.30.0 |      |
```

**Review by PyvesB — COMMENTED** [review]
Thanks for updating this! I did a pass over some of the minimal version requirements, which have since then changed.

**Review by PyvesB — APPROVED** [review]

**Review by jhgilbert — APPROVED** [review]

**Pull request merged**